JS: Health Application with  5th tab in the middle.

VardgesKarapetyan opened the pull request:
**Done**
Added the followings
* 5th tab in the middle with corresponding pages
* 'Add' button and 'delete' buttons with corresponding functionalities to Food tab (In ui-neumorphism style)
* dark mode button with corresponding functionality
* profile logo to Profile tab
* functionality to use localStorage for storing input values.

![LastLight](https://user-images.githubusercontent.com/60390251/120557595-38b6f500-c40f-11eb-9fc9-042abcc24666.png)
![LastDark](https://user-images.githubusercontent.com/60390251/120557617-41a7c680-c40f-11eb-9792-445db4e5afcd.png)

Commit:
Added book store

Commit:
Merge branch 'develop' of https://github.com/abrahamyan44/ITC2021 into develop

Commit:
Merge branch 'develop' of https://github.com/abrahamyan44/ITC2021 into develop

Commit:
Deleted book store

VardgesKarapetyan committed:
Deleted my Task

VardgesKarapetyan committed:
Health Application with  5th tab in the middle.

VardgesKarapetyan committed:
Added public package.

ghost commented:
Vardkez jan there are 2 tabs missing atthe end. You mast have that info in your **Remains** section. Navigation is optional so you can skip it.

manehambardzumyan commented on JS/Vardges/FitnesApplication/src/containers/MainContainer.jsx:
```diff
@@ -0,0 +1,116 @@
+    this.onSidebarClick = this.onSidebarClick.bind(this)
+  }
+
+  // get isHome() {
```
just comment
what if I type '//home' or just stupid thing

manehambardzumyan commented on JS/Vardges/FitnesApplication/public/index.html:
```diff
@@ -0,0 +1,110 @@
+      content='React component library designed on the "new skeuomorphism" or "neumorphism" UI/UX trend.'
+    />
+    <script type="text/javascript">
+      // Single Page Apps for GitHub Pages
```
I assume you should not have this part

manehambardzumyan commented on JS/Vardges/FitnesApplication/src/containers/MainContainer.jsx:
```diff
@@ -0,0 +1,116 @@
+                }`}
+              >
+              
+                <Switch>
```
See in code but done see in UI, where is this support to be

manehambardzumyan replied:
@VardgesKarapetyan I still don't see the switch and can not find in your screenshots as well

VardgesKarapetyan replied:
I have already deleted

manehambardzumyan replied:
@VardgesKarapetyan jan you did not answered me where is the switch/checkbox which changes to dark mode?

manehambardzumyan reviewed (changes requested):
Please few semantic issues and this part of homework is good !!!

VardgesKarapetyan committed:
Added last 2 tabs.Used localStorage for Storing input values.

VardgesKarapetyan committed:
Task with little changes

sqlite3* DB;
	std::string sql = "CREATE TABLE ndranik("
					"ID INT PRIMARY KEY	 NOT NULL, "
					"NAME		 TEXT NOT NULL, "
					"SURNAME		 TEXT	 NOT NULL, "
					"AGE		 INT	 NOT NULL, "
					"ADDRESS	 CHAR(50), "
					"SALARY		 REAL );";
	int exit = 0;
	exit = sqlite3_open("example.db", &DB);
	char* messaggeError;
	exit = sqlite3_exec(DB, sql.c_str(), NULL, 0, &messaggeError);

	if (exit != SQLITE_OK) {
		std::cerr << "Error Create Table" << std::endl;
		sqlite3_free(messaggeError);
	}
	else {
		std::cout << "Table created Successfully" << std::endl;
	}

VardgesKarapetyan commented on JS/Vardges/FitnesApplication/src/App.js:
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Button, Fab,IconButton ,RadioGroup} from 'ui-neumorphism'
```
Done

manehambardzumyan replied:
```suggestion
import { Button, Fab, IconButton, RadioGroup} from 'ui-neumorphism'
```

VardgesKarapetyan commented on JS/Vardges/FitnesApplication/src/MainPages/FitnessAppContainers/FoodPage/FoodList.jsx:
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+import FoodListItem from './FoodListItem'
+
+
```
Done

VardgesKarapetyan commented on JS/Vardges/FitnesApplication/src/MainPages/FitnessAppFunction.jsx:
```diff
@@ -0,0 +1,56 @@
+    ToggleButton,
+    ProgressCircular,
+} from 'ui-neumorphism'
+import HomePage1 from './FitnessAppContainers/HomePage1'
```
Done

VardgesKarapetyan commented on JS/Vardges/FitnesApplication/public/manifest.json:
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Done

VardgesKarapetyan commented on JS/Vardges/FitnesApplication/src/MainPages/index.js:
```diff
@@ -0,0 +1,4 @@
+// export { default as FitnessApp } from './FitnessApp'
+
+
+export {default as FitnessApp1} from './FitnessAppFunction'
```
Done

VardgesKarapetyan committed:
Added Profile image

manehambardzumyan commented on JS/Vardges/FitnesApplication/package.json:
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "project",
```
Fix the name

VardgesKarapetyan replied:
done

manehambardzumyan commented on JS/Vardges/FitnesApplication/src/MainPages/FitnessAppContainers/FoodPage/FoodForm.jsx:
```diff
@@ -0,0 +1,24 @@
+                setText("")
+            }}>
+                <input type="text" value={text}  maxLength={20} onChange={(ev) => {
+                    if(ev.target.value !== ' ')
```
Please explain what the following code does?
Please dont tell checks the IF, I am interested on logic its quite incomplete from my point of view

VardgesKarapetyan replied:
this prevents the user from being able to start the word with spaces

manehambardzumyan commented on JS/Vardges/FitnesApplication/src/MainPages/FitnessAppContainers/FoodPage/General.jsx:
```diff
@@ -0,0 +1,33 @@
+
+
+function Food({dark}) {
+   const [items, setTodos] = useLocalStorage(' ', [])
```
Why you send those arguments to 'useLocalStorage' function, could you please explain ? I am quite not sure that you understood what you did

VardgesKarapetyan replied:
I save the user's food list  in items object so that the list is not deleted after reloading page.

manehambardzumyan replied:
@VardgesKarapetyan jan and what is the key for that object?

manehambardzumyan commented on JS/Vardges/FitnesApplication/src/MainPages/FitnessAppContainers/FoodlIst.jsx:
```diff
@@ -0,0 +1,96 @@
+                            Food List
+              </H4>
+                        {/* <IconButton
+                size='small'
```
Please delete

VardgesKarapetyan replied:
done

manehambardzumyan commented on JS/Vardges/FitnesApplication/src/MainPages/FitnessAppContainers/FoodPage/General.jsx:
```diff
@@ -0,0 +1,33 @@
+    return (
+        <div>
+            <FoodForm dark={dark} onAdd={(text) => {
+                if (items.length < 10 && text.length !== 0) {
```
Why 10 ? Please explain what the code does (Logic)

VardgesKarapetyan replied:
I just limited it here
the size of the user food list

VardgesKarapetyan committed:
FitnessApplication with several changes

VardgesKarapetyan committed:
FitnessApplication last version with changes
